### PR TITLE
Update docs to nightly

### PR DESCRIPTION
**Pull request by 5chdn**
[DO NOT MERGE]

Just testing the docs script. :tada:

### CODE REVIEW

File: JSONRPC-eth-module.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
----
-title: The `eth` Module
----
+# The `eth` Module
```
Comment (5chdn) [inline JSONRPC-eth-module.md]: Is this an issue? @Tbaut

File: JSONRPC-db-module.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# The `db` Module
```
Comment (5chdn) [inline JSONRPC-db-module.md]: what is this?

File: JSONRPC-eth_pubsub-module.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+# The `eth_pubsub` Module
```
Comment (5chdn) [inline JSONRPC-eth_pubsub-module.md]: new file?

**Comment by 5chdn** [comment]
why are there conflicting files?

File: JSONRPC-parity_set-module.md [hunk 4]
```diff
@@ -324,7 +353,7 @@ Response
 
 ### parity_setEngineSigner
 
-Sets an authority account for signing consensus messages. For more information check the [Proof of Authority Chains](Proof-of-Authority-Chains.md) page.
+Sets an authority account for signing consensus messages. For more information check the [[Proof of Authority Chains]] page.
```
Comment (5chdn) [inline JSONRPC-parity_set-module.md]: will that link work after merge?

**Comment by Tbaut** [comment]
I've been changing the scripts slightly recently to make sure we have less weird things: https://github.com/parity-js/jsonrpc/tree/Tbaut-patch-update-to-new-docs

**Comment by 5chdn** [comment]
Ah I see!

**Comment by Tbaut** [comment]
I'll close this one, I'm looking at the diffs on my branch and will open a PR when the json-rpc updates are accepted.

**Pull request closed without merging**